unit_tests/sources/declarative/resolvers/test_http_components_resolver.py

```diff
@@ -0,0 +1,189 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+import json
+from unittest.mock import MagicMock
+
+import pytest
+
+from airbyte_cdk.models import Type
+from airbyte_cdk.sources.declarative.concurrent_declarative_source import (
+    ConcurrentDeclarativeSource,
+)
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.resolvers import (
+    ComponentMappingDefinition,
+    HttpComponentsResolver,
+)
+from airbyte_cdk.sources.embedded.catalog import (
+    to_configured_catalog,
+    to_configured_stream,
+)
+from airbyte_cdk.test.mock_http import HttpMocker, HttpRequest, HttpResponse
+
+_CONFIG = {"start_date": "2024-07-01T00:00:00.000Z"}
+
+_MANIFEST = {
+    "version": "5.0.0",
```

Comment:
Weird that this would work — CDK version seems to be around 6.7 nowdays?

Reply:
Updated to 6.7.0. This manifest is used to test manifest schema validation, so the actual values do not affect anything.